src/workerd/api/node/zlib-util.h

```diff
@@ -90,15 +90,14 @@ struct CompressionError {
   int err;
 };
 
-// TODO(soon): See if RAII support can be added directly to this class, and we can mark it final
-class ZlibContext {
+class ZlibContext final {
 public:
   explicit ZlibContext(ZlibMode _mode): mode(_mode) {}
   ZlibContext() = default;
+  ~ZlibContext();
```

Comment:
By convention we use `noexcept(false)` for destructors.

Reply:
Yes, actually I realized that we don't even need them...